scripts/flow.sh

```diff
@@ -28,12 +28,30 @@ apply() {
 }
 
 delete() {
+    echo "Deleting Load Balancers"
```

Comment:
Is this necessary if the CLI is already going to recursively delete entities for `environment delete`?

Reply:
well that doesn't exist right now so this is filling in for the itermin